src/compiler/scala/tools/nsc/classpath/DirectoryClassPath.scala

```diff
@@ -145,28 +145,52 @@ object JrtClassPath {
 
       val currentMajorVersion: Int = JDK9Reflectors.runtimeVersionMajor(JDK9Reflectors.runtimeVersion()).intValue()
       release match {
-        case Some(v) if v.toInt < currentMajorVersion =>
-          try {
-            val ctSym = Paths.get(javaHome).resolve("lib").resolve("ct.sym")
-            if (Files.notExists(ctSym)) None
-            else {
-              val classPath = ctSymClassPathCache.getOrCreate(v, ctSym :: Nil, () => new CtSymClassPath(ctSym, v.toInt), closeableRegistry, true)
-              Some(classPath)
-            }
-          } catch {
-            case _: Throwable => None
+        case Some(version) if version.toInt < currentMajorVersion =>
+          val ct = createCt(version, closeableRegistry)
+          unsafe match {
+            case Some(pkgs) =>
+              val open = if (pkgs.isEmpty) List("sun.misc") else pkgs
```

Comment:
Maybe skip the default? It gives more work for documentation.

Reply:
When I updated the PR comment, I realized no one will use `-Yunsafe:some.weird.pkg`; they will only use `-Yunsafe`.